src/consensus/pbft_manager.cpp

```diff
@@ -126,6 +126,87 @@ void PbftManager::run() {
   // Initialize PBFT status
   initialState_();
 
+  continuousOperation_();
+}
+
+// Only to be used for tests...
```

Comment:
I think resume() is not for test, and resumeSingleState() is for test only?

Reply:
Why do you say that, where is it used besides in the test?  I just added it myself for this test (and future tests).  ha.